web/public/js/controllers/user-profile-controller.js

```diff
@@ -235,13 +236,14 @@ function cdUserProfileCtrl($scope, $rootScope, $state, $window, auth, cdUsersSer
       alertService.showError( $translate.instant('Error loading Dojos') + ' ' + err);
     });
 
-    cdUsersService.loadDojoAdminsForUserPromise(loggedInUser.data.id).then(
-      function(champions){
-        $scope.loggedInUserIsMemberOfDojoChampion = !!(_.find(champions, function (championForUser) {
-          return championForUser.id === profileUserId;
-        }));
-      }
-    );
+    cdUsersService.loadChildrenForUser($stateParams.userId, function (children) {
+      $scope.profile.resolvedChildren = children;
+    });
+
+    cdUsersService.loadParentsForUserPromise($stateParams.userId)
```

Comment:
The resolve for parentsForUser is still being used along with this.

Same point as well as regarding `$scope.profile.parents.length`.

I'd also like to see a loading animation in these sections if we need to fetch children or parents.

Also, this uses a Promise, children uses a callback. Probably should use Promise for children as well for consistency.